plugins/inputs/snmp/snmp.go

```diff
@@ -679,6 +680,7 @@ func fieldConvert(conv string, v interface{}) (interface{}, error) {
 		case float32:
 			v = float64(vt) / math.Pow10(d)
 		case float64:
+			//nolint:unconvert // Conversion may be needed for float64 https://github.com/mdempsky/unconvert/issues/40
 			v = float64(vt) / math.Pow10(d)
```

Comment:
This is because conversion from float to float is sometimes used to prevent the compiler's Fused Multiply Add (FMA) instruction selection to avoid rounding errors. I saw places in Telegraf's code after I removed `float64(float64Var)` conversion tests started to fail.

More details: 
https://stackoverflow.com/questions/63544888/delete-conversion-changes-semantics

It seems that in this case removing conversion works.